<launch>
    <param name="top_lidar_topic" value="/cti/sensor/rslidar/PointCloud2"/>
    <param name="front_lidar_topic" value="/cti/sensor/rslidar/Bpearl_PointCloud2"/>
    <param name="front_scans" value="32"/>
    <node pkg="calibration" name="calibration" type="calibration_node" output="screen"/>
</launch>
